Playground/RxSwiftExtPlayground.playground/Pages/mergeWith.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,37 @@
+/*:
+ > # IMPORTANT: To use `RxSwiftExtPlayground.playground`, please:
+
+ 1. Make sure you have [Carthage](https://github.com/Carthage/Carthage) installed
+ 1. Fetch Carthage dependencies from shell: `carthage bootstrap --platform ios`
+ 1. Build scheme `RxSwiftExtPlayground` scheme for a simulator target
+ 1. Choose `View > Show Debug Area`
+ */
+
+//: [Previous](@previous)
+
+import RxSwift
+import RxSwiftExt
+
+/*:
+ ## merge(with:)
+ Merges elements from observable sequences into a single observable sequence.
+
+ */
+example("merge values") {
```

Comment:
example title should be the name of the operator

Reply:
Fixed